apps/ios/Shared/Views/Chat/ChatItem/CIMetaView.swift

```diff
@@ -45,7 +49,7 @@ func ciMetaText(
     _ meta: CIMeta,
     chatTTL: Int?,
     encrypted: Bool?,
-    color: Color = .clear,
+    color: Color? = .clear,
```

Comment:
`Color?` -  passing nil here means that `.foregroundColor` modifier gets completely omitted.